clients/src/main/java/org/apache/kafka/clients/consumer/KafkaConsumer.java

```diff
@@ -1385,6 +1385,10 @@ public void seekToEnd(Collection<TopicPartition> partitions) {
 
     /**
      * Get the offset of the <i>next record</i> that will be fetched (if a record with that offset exists).
+     * This method will issue a remote call to the server.
```

Comment:
This method _may_ issue a remote call, but only if there is no current position for the partition.